smaug/python/ops/activation_ops_test.py

```diff
@@ -5,7 +5,7 @@
 import unittest
 import numpy as np
 
-from smaug.python import smaug_test
+from smaug.python.smaug_test import *
```

Comment:
We really should avoid import * as much as possible because it makes it much more difficult to track down the source of a function or object definition. We can rename the final library if need be though (import long_foo_lib as foo), but the confusion of searching for the definition is not worth the few characters written. Remember that code is read much more than it is written, so it should be written to be easy to read.

To me, it looks like this commit is mostly about moving get_node from the testcase class to a standalone function. If you want to reduce boilerplate, why not put get_node as a member function of Graph? So instead of doing `get_node(test_graph.graph, node_name)`, you can just say `test_graph.get_node(node_name)`.

Reply:
Yeah, I should have that mentality for a better practice. I will have a separate PR for purging all the *.

The get_node method is added to Graph in this PR.

Reply:
Well I changed my mind. Let's purge all the import * in this PR. Just uploaded a commit for that.